.ci/openshift-ci/build/Dockerfile

```diff
@@ -24,17 +24,16 @@ ADD . /che-plugin-registry
 WORKDIR /che-plugin-registry
 
 # Install build tools
-RUN yum update -y && yum upgrade -y
-RUN yum install git -y 
-RUN yum install openssh -y
-RUN curl -sL https://rpm.nodesource.com/setup_14.x | bash -
-RUN yum install nodejs -y
-RUN npm install --global yarn
+RUN yum update -y && yum upgrade -y &&\
+    yum install git -y &&\
+    yum install openssh -y &&\
+    curl -sL https://rpm.nodesource.com/setup_14.x | bash - &&\
+    yum install nodejs -y &&\
+    npm install --global yarn
 
 # Build repo
-RUN yarn && \
-    yarn --cwd tools/build build && \
-    eval node tools/build/lib/entrypoint.js --output-folder:"output"
+
+RUN ./build.sh --skip-oci-image
```

Comment:
Sorry I forgot to mention the other options which may be useful: `SKIP_TEST=true SKIP_FORMAT=true SKIP_LINT=true ./build.sh --skip-oci-image --skip-digest-generation`